### PR TITLE
docs: github pull request template update

### PR DESCRIPTION
**Pull request by danielo515**
# Description

just updated the PR template

## How Has This Been Tested?

is just docs

### CODE REVIEW

**Comment by danielo515** [comment]
@LostNeophyte as you suggested

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,11 +1,13 @@
 <!-- This won't be rendered!
 [CHECKLIST]
 I prefixed the title with one of the following tags:
- - feature: for feature addition / improvements
+ - feat: for feature addition / improvements
  - fix: when fixing a functionality
  - refactor: when moving code without adding any functionality
  - doc: on documentation updates
```
Comment (LostNeophyte) [inline .github/pull_request_template.md:7]:
```suggestion
 - docs: on documentation updates
```

**Review by LostNeophyte — CHANGES_REQUESTED** [review]
Thanks :)

**Review by LostNeophyte — APPROVED** [review]
LGTM :)

**Comment by LostNeophyte** [comment]
Sorry I broke this PR, can you make a new one targeted at master

**Comment by LostNeophyte** [comment]
Alright I did it here https://github.com/LunarVim/LunarVim/pull/3512, sorry again, i added wrong remote while rebasing (lunarvim.org) :facepalm:

**Comment by danielo515** [comment]
> Alright I did it here #3512, sorry again, i added wrong remote while rebasing (lunarvim.org) 🤦 i should have realized something was wrong by the 1000 commits it wanted me to rebase 🤦

Thanks for taking care

**Pull request closed without merging**